lib/token.js

```diff
@@ -118,9 +118,10 @@ function config () {
     })
   } else {
     conf = conf.concat({ auth: {} })
-    conf.auth = {}
   }
-  if (conf.otp) conf.auth.otp = conf.otp
+  if (conf.otp) {
```

Comment:
This second bit is actually not what you want.  It means that `npm token <whatever> --otp=123456` will throw away the auth.

Since we've already set `auth` to an object, we can modify _that object_, because it's not a key on the figgy pudding container.

I'll land this, but without that part, just posting this comment for posterity :)